src/pae/mka_secy.c

```diff
@@ -172,6 +172,7 @@ static uint8_t CheckNextRxSAFree(t_MKA_receive_sa const * sa)
 void MKA_SECY_Init(t_MKA_bus bus)
 {
     t_MKA_secy * const ctx = &mka_secy[bus];
+    t_MKA_bus_config const * const bus_cfg = &MKA_active_buses_config[bus];
```

Comment:
please use the same variable name as in the rest of the code, "bus" instead of "bus_cfg"